Makefile

```diff
@@ -49,7 +49,7 @@ help:
 
 .PHONY: inplace
 inplace: clean
-	python3 -m pip install -e --global-option="build_ext" --global-option="--inplace" .
```

Comment:
I'm not a fan of inplace anyway, I'd rather this option went away as it's broken when you have virtual envs or more than one python you're testing. It's safer to have things that always work than ones that sometimes work.

Reply:
I don't use it either.